object_database/web/cells/non_display/key_action.py

```diff
@@ -0,0 +1,41 @@
+"""KeyAction Cell
+
+Listens key-combo press events from the UI"""
+from ..cells import Cell
+
+
+class KeyAction(Cell):
+    """
+    Parameters
+    __________
+    keyCmd str: A string of a key combo like `Alt+i`, `Meta+D`
+                or event single keys like `X`.
+    callback func: A function that will be called with the
+                event response dictionary data as the sole arg.
```

Comment:
can we document the contents of this dictionary? I'm not sure what would be in there or how I'd use it as a python programmer.

Reply:
I'm not sure this is clear, but the dictionary data is requested when the Python user instantiates the `KeyAction` Cell. By default it will just be an empty dictionary since by default you don't pass any names of data you want from the browser.